pinecone/client.go

```diff
@@ -24,15 +24,16 @@ import (
 // Client holds the parameters for connecting to the Pinecone service. It is returned by the NewClient and NewClientBase
 // functions. To use Client, first build the parameters of the request using NewClientParams (or NewClientBaseParams).
 // Then, pass those parameters into the NewClient (or NewClientBase) function to create a new Client object.
-// Once instantiated, you can use Client to execute control plane API requests (e.g. create an Index, list Indexes,
-// etc.). Read more about different control plane API routes at [docs.pinecone.io/reference/api].
+// Once instantiated, you can use Client to execute Pinecone API requests (e.g. create an Index, list Indexes,
+// etc.), and Inference API requests. Read more about different Pinecone API routes at [docs.pinecone.io/reference/api].
 //
 // Note: Client methods are safe for concurrent use.
 //
 // Fields:
-//   - headers: An optional map of additional HTTP headers to include in each API request to the control plane,
-//     provided through NewClientParams.Headers or NewClientBaseParams.Headers.
-//   - restClient: Optional underlying *http.Client object used to communicate with the Pinecone control plane API,
+//   - Inference: An InferenceService object that exposes methods for interacting with the Pinecone [Inference API].
+//   - headers: An optional map of additional HTTP headers to include in each API request, provided through
```

Comment:
Total nit: "additional" seems redundant here, considering it's an optional field